omim2obo/main.py

```diff
@@ -139,6 +139,7 @@ def omim2obo(use_cache: bool = False):
     # - Non-OMIM triples
     graph.add((URIRef('http://purl.obolibrary.org/obo/mondo/omim.owl'), RDF.type, OWL.Ontology))
     graph.add((URIRef('http://www.geneontology.org/formats/oboInOwl#hasSynonymType'), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef('http://purl.obolibrary.org/obo/mondo#omim_included'), RDF.type, OWL.AnnotationProperty))
```

Comment:
The second (data property) solution is very far from correct syntax.

@twhetzel and I considered the first option, but again, it is wrong, but for another reason: included "entities" are not synonyms. In some cases it may be ok to add "relatedSynonym", but most of the time not even that is correct. So, its better to not at all at included enities as synonyms!